### ISSUE TITLE
cake-contrib user on NuGet

### ISSUE THREAD

**Issue by gep13** [SharpeRAD/Cake.Powershell#9]
First of all, I wanted to thank you for adding to the Cake community by adding this addin.

I was just wondering if you had seen this blog post:

http://cakebuild.net/blog/2016/08/cake-contribution-organization

We are currently going through a process of increasing the visibility of addins, and also trying to ensure their long term maintainability.

To that end, we are asking addin creators to add the cake-contrib user on NuGet as a co-owner (this can be done through the NuGet website by clicking on `Manage Owners` on the package page).

Would you be interested in doing this for Cake.PowerShell?  If you have any questions about this, please let me know.  There was some initial concern that the Cake Team were trying to "take over" packages, and that couldn't be further from the truth, and if you have this concern, or others, I would like to address them.

Thanks!

**Comment by gep13** [comment]
@SharpeRAD was just wondering if you had given any thought to this for this addin, and your others.  Thanks!

**Comment by SharpeRAD** [comment]
Yea I will end up adding it, just huffing and puffing for a bit as I didn't like that it began with someone not getting a reply in 2days. Honestly don't know how you guys juggle work / family and Github, just watching Cakes notifications makes me tired 😪

**Comment by gep13** [comment]
>@SharpeRAD said...
Yea I will end up adding it

That is great news, glad to have your support!

>@SharpeRAD said...
just huffing and puffing for a bit as I didn't like that it began with someone not getting a reply in 2days

That might be how the `conversation` started around this, but that certainly isn't why we have implemented these suggestions, and certainly that wouldn't be a driving force behind anyone on the Cake team making an intervention on a community addin.  As I have discussed with @agc93 at length, if an addin is perceived to has fallen out of maintenance by a community member, and a Cake Team member is asked to step in we will do everything in our power, i.e. GitHub, Twitter, Email, Smoke Signals, to get in touch with the original creator of the addin, we won't simply take over.

>@SharpeRAD said...
Honestly don't know how you guys juggle work / family and Github, just watching Cakes notifications makes me tired 😪

Exactly!  We don't `want` more work, but if, and only if, an addin has fallen out of maintenance, we want the ability to push a new package version, hence the request to add the cake-contrib user as a co-owner of the package on NuGet.org, in order to prevent duplication of the packages.

As I mentioned, if you wanted to chat about this more, happy to arrange a skype call or something at some point to discuss further.
